core/src/main/java/com/linecorp/armeria/common/http/PathParamExtractor.java

```diff
@@ -26,38 +28,41 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import javax.annotation.Nullable;
-
 import com.google.common.collect.ImmutableSet;
 
 /**
- * A bindable path. It holds three things:
+ * A path param extractor. It holds three things:
  * <ul>
- *   <li> A regex-compiled form of the path. It is used for matching and binding.
+ *   <li> A regex-compiled form of the path. It is used for matching and extracting.
  *   <li> A skeleton of the path. It is used for duplication detecting.
  *   <li> A set of variables declared in the path.
```

Comment:
Would you mind:

- Removing the leading space for each list item and
- Closing the `<li>` tag

.. although it's not your change strictly speaking?